src/Microsoft.ML.DnnImageFeaturizer.ResNet101/Microsoft.ML.DnnImageFeaturizer.ResNet101.csproj

```diff
@@ -16,9 +16,26 @@
     <ProjectReference Include="..\Microsoft.ML.Data\Microsoft.ML.Data.csproj" >
       <PrivateAssets>all</PrivateAssets>
     </ProjectReference>
+
+    <PackageReference Include="Microsoft.ML.Assets" Version="$(MicrosoftMLAssetsVersions)" PrivateAssets="all" GeneratePathProperty="true"/>
   </ItemGroup>
 
   <ItemGroup>
     <Content Include="$(RepoRoot)eng\pkg\DnnImageFeaturizer.props" Pack="true" PackagePath="build\netstandard2.0\$(MSBuildProjectName).props" />
   </ItemGroup>
+
+<Target Name="CallCopyDnnOnnxModels" BeforeTargets="Build" >
+    <ItemGroup>
+      <ModelFileCopy Include="$(PkgMicrosoft_ML_Assets)contentFiles\any\any\ResNetPrepOnnx\ResNetPreprocess.onnx"
+      TargetPath="$(PackageAssetsPath)\Microsoft.ML.DnnImageFeaturizer.ResNet101\tools\DnnImageModels\ResNetPrepOnnx\ResNetPreprocess.onnx" />
```

Comment:
Perhaps not as part of this, but ResNet101 could use a unit test. Last I tried ResNet18 worked, but 101 failed.